redisinsight/ui/src/telemetry/telemetryUtils.ts

```diff
@@ -19,6 +20,8 @@ import {
 import { TelemetryEvent } from './events'
 import { checkIsAnalyticsGranted } from './checkAnalytics'
 
+export const getProvider = (): string => get(store.getState(), 'connections.instances.connectedInstance.provider', '')
```

Comment:
it is better to have `undefined` instead of `''` (empty string). In this case provider field won't be sent at all

Reply:
@ArtemHoruzhenko, fixed to be undefined by default